src/styles/deserializers.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {degreesToRadians, radiansToDegrees} from './conversions.js';
+import {IdentNode, parseExpressions} from './parsers.js';
+
+
+/**
+ * Parses an angle CSS number string and deserializes it as a number with the
+ * given unit type. For example:
+ *
+ * deserializeAngle('1rad', 'rad'); // Returns 1
+ * deserializeAngle('1rad', 'deg'); // Returns 57.29577951308232
+ * deserializeAngle('180deg', 'rad'); // Returns 3.141592653589793
+ */
+export const deserializeAngle =
+    (angleString: string, unit: 'deg'|'rad' = 'deg'): number|null => {
```

Comment:
Perhaps we should remove the default to enforce descriptive units where it is used.

Reply:
SGTM